packages/zoe/src/contracts/priceAggregator.js

```diff
@@ -44,7 +45,9 @@ import '../../tools/types.js';
  * }} privateArgs
  */
 const start = async (zcf, privateArgs) => {
-  const { timer, POLL_INTERVAL, brandIn, brandOut } = zcf.getTerms();
+  const { timer, POLL_INTERVAL, brandIn, brandOut, unitAmountIn } =
```

Comment:
hm... contract creator has to provide `unitAmountIn`. A little inconvenient, perhaps, but certainly on the safe side of "don't use displayInfo for anything other than display."